sirepo/pkcli/roles.py

```diff
@@ -21,29 +21,22 @@ def add(uid_or_email, *roles, expiration=None):
         *roles (str): The roles to assign to the user
         expiration (int): Days until expiration
     """
-    with _parse_args(uid_or_email, roles) as qcall:
-        qcall.auth_db.model("UserRole").add_roles(
-            roles, expiration=_parse_expiration(expiration)
+    with _parse_args(uid_or_email, roles, expiration) as a:
+        a.qcall.auth_db.model("UserRole").add_roles(
+            a.roles,
+            uid=a.uid,
+            expiration=a.expiration,
         )
 
 
-def add_or_update(uid_or_email, *roles, expiration=None):
-    """Add roles or update expirations to existing roles for user.
-    Args:
-        uid_or_email (str): Uid or email of the user
-        *roles (str): The roles to assign to the user
-        expiration (int): Days until expiration
-    """
-    with _parse_args(uid_or_email, roles) as qcall:
-        for r in roles:
-            qcall.auth_db.model("UserRole").add_role_or_update_expiration(
-                r, _parse_expiration(expiration)
-            )
+def add_plan(uid_or_email, plan, expiration=None):
+    with _parse_args(uid_or_email, [plan], expiration) as a:
+        qcall.auth_db.model("UserRole").add_plan(a.roles[0], a.uid, expiration)
```

Comment:
There is a bug when a user is create from admin, possibly in ```add_plan()```. The user ends up with a list of single letter roles:
```
$ sirepo admin create-user vagrant@localhost.localdomain "test user"
$ sirepo roles list-with-expiration vagrant@localhost.localdomain
{'role': 'basic', 'expiration': None}
{'role': 'e', 'expiration': None}
{'role': 'i', 'expiration': None}
{'role': 'm', 'expiration': None}
{'role': 'p', 'expiration': None}
{'role': 'r', 'expiration': None}
{'role': 'u', 'expiration': None}
{'role': 'user', 'expiration': None}
```

Reply:
a16a70a